docs/publishing-your-site.md

```diff
@@ -4,6 +4,13 @@ The great thing about hosting project documentation in a `git` repository is
 the ability to deploy it automatically when new changes are pushed. MkDocs
 makes this ridiculously simple.
 
+???+ warning
+
+    Please note that the container image with which mkdocs-material is delivered is not an image for
```

Comment:
IMHO, this is misleading, it sounds like you should not build the documentation, which is what the image is for. Also, I'm not sure whether this page is the right choice to add this admonition, it might be rather confusing.

Reply:
Maybe rather under the Docker installation instructions in the getting started guide? Here's a draft:

> Please note that the Docker container is optimized for local previewing, not for deployments. The reason for this is that the web server that MkDocs provides for the live preview functionality is not production-grade, and might have security issues.

There might be some note on the MkDocs docs that mentions this, but I'm having a hard time finding it. There ar, however, a lot of issues where the maintainers of MkDocs have pointed this out to users trying to just run `mkdocs serve` in production.